### DISCUSSION TITLE
Welcome to Laragine Discussions!

### DISCUSSION THREAD

**Discussion by mahmoodahmad100** [yepwoo/laragine#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.
  
  Here is also a quick short video about Laragine:
  https://www.youtube.com/watch?v=5klckaSiM6g

**Comment by mahmoodahmad100** [comment]
Hi @mitulgolakiya I happen to checkout your profile and it's nice to know that you have created a lot of amazing Laravel packages, I just want to let you know about Laragine as you have said "I really enjoy working in Laravel and Open-source technologies" so I just want to share it with you as this might be something interesting to contribute to. thanks 😃

**Reply by mitulgolakiya** [reply]
Hi @mahmoodahmad100 thank you for tagging me. I've just tried to give it a hand. Really nice package to start with. Here are my initial thoughts and some idea that may make the package better.

1. More Docs on creating views and layout. It seems it's completely missing. It took my 30 mins to get my first view created and loaded
2. While creating a Unit, if it can also create all the views with basic CRUD then it will be great
3. There are too many installation steps (I made this same mistake in my CRUD generators, which I fixed recently). Nobody likes the lengthly and long installation steps. so if you can create one command that can automate installation steps like logging, validation, middleware configuration directly into the install command, then it will be great.

Again, these are my personal suggestions, but you know the best since you know all the plans and upcoming things of the package.

But at first impression, it can be a great package if proper efforts are placed.

**Reply by mahmoodahmad100** [reply]
Thanks @mitulgolakiya a lot  for the valuable thoughts, I really appreciate it, we wanted to make tutorial (YouTube) on how to use the package but looks like we will have to make some changes first as you have suggested.

Thanks a lot for the amazing suggestions!

**Comment by icarojobs** [comment]
Hello for everyone!

I'd like to know if Laragine works fine with Laravel Livewire? If yes, I'm starting to use it today!

**Reply by mahmoodahmad100** [reply]
Hi, thanks for your comment!

Yes you can use any packages with Laragine including Laravel Livewire, that shouldn't make any conflicts
